addons/website/static/tests/interactions/snippets/form.test.js

```diff
@@ -13,7 +13,10 @@ describe.current.tags("interaction_dev");
 function checkField(inputEl, isVisible, hasError) {
     const fieldEl = inputEl.closest(".s_website_form_field");
     isVisible ? expect(fieldEl).not.toHaveClass("d-none") : expect(fieldEl).toHaveClass("d-none");
-    isVisible ? expect(inputEl).toBeEnabled() : expect(inputEl).not.toBeEnabled();
+    // Inputs required for the model are never disabled.
+    if (!fieldEl.matches(".s_website_form_model_required")) {
+        isVisible ? expect(inputEl).toBeEnabled() : expect(inputEl).not.toBeEnabled();
+    }
```

Comment:
Now that model-required fields are always enabled, do not check whether they are `disabled or enabled` to consider them `hidden or visible`.